apps/condo/domains/miniapp/schema/B2BAppAccessRightSet.test.js

```diff
@@ -1360,4 +1468,73 @@ describe('B2BApp permissions for service user', () => {
 
         })
     })
+
+    describe('B2BAccessTokens domain', () => {
```

Comment:
```suggestion
    describe('B2BAccessTokens hooks test', () => {
```

Reply:
This test is testing that rights restrictions from B2BAppAccessRightSet are working, so these must be here